actionpack/lib/action_dispatch/middleware/templates/rescues/javascript_layout.erb

```diff
@@ -0,0 +1 @@
+alert("<%= j yield %>");
```

Comment:
Modals are intrusive and annoying by design. If I was debugging a tricky error I wouldn't want it to keep popping up in a modal. The console would be a much more fitting place for error. Would you consider changing this to `console.error` or `throw new Error`?